tests/playwright/pages/editor-page.ts

```diff
@@ -481,15 +481,84 @@ export default class EditorPage extends BasePage {
 		await this.page.locator( '.elementor-control-type-code textarea' ).fill( css );
 	}
 
-	async changeUiTheme( uiMode: string ) {
+	/**
+	 * Whether the Top Bar is active or not.
+	 *
+	 * @return {boolean}
+	 */
+	async hasTopBar() {
+		return await this.page.locator( EditorSelectors.topBar.wrapper ).isVisible();
+	}
+
+	/**
+	 * Open the menu panel.
+	 */
+	async openMenuPanel() {
+		await this.page.locator( EditorSelectors.panels.menu.footerButton ).click();
+		await this.page.waitForSelector( EditorSelectors.panels.menu.wrapper );
+	}
+
+	/**
+	 * Open the elements/widgets panel.
+	 */
+	async openElementsPanel() {
+		const hasTopBar = await this.hasTopBar();
+
+		if ( hasTopBar ) {
+			await this.page.locator( EditorSelectors.topBar.wrapper ).getByText( 'Elements' ).click();
+		} else {
+			await this.page.locator( EditorSelectors.panels.elements.footerButton ).click();
+		}
+
+		await this.page.waitForSelector( EditorSelectors.panels.elements.wrapper );
```

Comment:
same here:
await this.page.locator( EditorSelectors.panels.elements.wrapper ).waitFor()